crates/refunder/src/submitter.rs

```diff
@@ -22,28 +22,27 @@ use shared::{
     submitter_constants::{TX_ALREADY_KNOWN, TX_ALREADY_MINED},
 };
 
-// Max gas used for submitting transactions
-// If the gas price is higher than this value,
-// the service will temporarily not refund users
+// Max gas price used for submitting transactions
 const MAX_GAS_PRICE: u64 = 800_000_000_000;
 
 // The gas price buffer determines the gas price buffer used to
 // send out EIP1559 txs.
 // Example: If the prevailing gas is 10Gwei and the buffer factor is 1.20
 // then the gas_price used will be 12.
 const GAS_PRICE_BUFFER_FACTOR: f64 = 1.3;
-// Max priority fee that the refunder is willing to pay. (=2 Gwei)
-const MAX_PRIORITY_FEE_TIP: u64 = 2_000_000_000;
-// In order to resubmit a new tx with the same nonce, the gas price needs to
-// be increased by at least 10 percent. We increase it by 12 percent
+// Starting priority fee that the refunder is willing to pay. (=2 Gwei)
+const START_PRIORITY_FEE_TIP: u64 = 2_000_000_000;
+
+// In order to resubmit a new tx with the same nonce, the gas tip and
+// max_fee_per_gas needs to be increased by at least 10 percent.
 const GAS_PRICE_BUMP: f64 = 1.12;
```

Comment:
Just for the consistency, I would change this factor to `1.125` since we are using the same factor in `solver` crate for the same thing.